doc/adr/0005-design-of-es-documents.md

```diff
@@ -0,0 +1,18 @@
+# 5. Design of ES documents
+
+* Status: accepted
+* Deciders: Fabien Boucher
+* Date: 2019-12-02
+
+## Context and Problem Statement
+
+We want to support most of the existing Code Review systems. Monocle queries should be agnostic regarding the data source.
+
+## Considered Options
+
+* A crawler by data source and an unique and generic data schema.
```

Comment:
Could you describe the different events and change here?

Reply:
Done